tex/generic/pgf/systemlayer/pgfsys.code.tex

```diff
@@ -862,6 +862,10 @@
 % state. Parameters #2 to #7 specify a transformation cm that should
 % be applied to the mask before it is used.
 
+\def\pgfsys@@clipfading{}
```

Comment:
If it is not used in other drivers, please make it
```suggestion
\def\pgfsys@clipfading{\pgf@sys@fail{clipfading}}
```
and implement it as a no-op in the respective driver files so it is not used accidentally (and use a single `@`).  It also needs documentation in `pgfmanual-en-pgfsys-commands.tex`.

Reply:
I dropped the second `@` from the name. I also documented it in `pgfmanual-en-pgfsys-commands.tex`. However, I am unsure about adding a void `\pgfsys@clipfading` definition to each and every driver file besides `pgfsys-dvips.def`. Would you perhaps accept the concept of a "default implementation" that doesn't need be re-implemented for other drivers, except the already existing implementation in `pgfsys-dvips.def`? This approach, I think, has already been taken for a number of other system-layer commands.